src/chromatix/field.py

```diff
@@ -225,6 +235,55 @@ def conj(self) -> Array:
         """conjugate of the complex field, as a field of the same shape."""
         return self.replace(u=jnp.conj(self.u))
 
+    @property
+    def spatial_limits(self) -> Tuple[Tuple[float, float], Tuple[float, float]]:
+        """
+        Return the spatial limits of the field: (y_min, y_max), (x_min, x_max).
+        """
+        return (float(self.grid[0].min()), float(self.grid[0].max())), (
+            float(self.grid[1].min()),
+            float(self.grid[1].max()),
+        )
+
+    def shift(self, shift_yx: Union[float, Array]) -> Field:
```

Comment:
@diptodip I added this method to set the shift argument (and essentially shift the whole field)

Reply:
This is great, but we do also have a `shift` function at the bottom of this file that might get confusing. That function I think actually crops and pads the array itself rather than defining a shift on the grid which is what we're doing here. Also, I've been trying to keep functions that operate on `Field`s as separate functions like the `pad`, `crop `and `shift `functions below, leaving only properties and math operators on the `Field` object. Could you rename either this function or the other one, and also make this a standalone function like the functions below?